Client/src/components/App.js

```diff
@@ -1,13 +1,55 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import Login from './contanier/Auth';
 import Mobile from './Layout/Mobile';
-// import Descktop from './Layout/Desktop';
-
-export default function App() {
-  return (
-    <div>
-      <Mobile />
-      {/* <Descktop /> */}
-    </div>
-  )
+import Descktop from './Layout/Desktop';
+import axios from 'axios';
+
+export default class App extends Component {
+  state = {
+    auth: false,
```

Comment:
@AbdallahAmmar96 I think changing the name of `auth` to `hasLoaded` would make the code a little easier to understand